src/hooks/useTasks.ts

```diff
@@ -0,0 +1,16 @@
+// src/hooks/useTasks.ts
+import { useTaskContext } from '../context/TaskContext';
+
+const useTasks = () => {
+    const { tasks, addTask, updateTask, deleteTask, toggleComplete } = useTaskContext();
+
+    return {
+        tasks,
+        addTask,
+        updateTask,
+        deleteTask,
+        toggleComplete,
+    };
+};
```

Comment:
Do we need this hook? I mean I can only see the we are just returning values from context and no additional logic as such.

Reply:
removed